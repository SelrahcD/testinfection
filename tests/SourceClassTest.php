<?php

namespace App;

use ApprovalTests\CombinationApprovals;
use PHPUnit\Framework\TestCase;

class SourceClassTest extends TestCase
{
    public function test_hello()
    {
        $firstnames = ['Charles', 'Maks'];

        CombinationApprovals::verifyAllCombinations1(function ($firstname) {
            $sourceClass = new SourceClass();
            return $sourceClass->hello($firstname);
        }, $firstnames);

    }
}
