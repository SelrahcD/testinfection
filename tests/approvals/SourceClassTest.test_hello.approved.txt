[Charles] => hello Charles
[Maks] => hello Maks
